WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -43,11 +44,14 @@ class ReaderPostUiStateBuilder @Inject constructor(
     private val dateTimeUtilsWrapper: DateTimeUtilsWrapper,
     private val readerImageScannerProvider: ReaderImageScannerProvider,
     private val readerUtilsWrapper: ReaderUtilsWrapper,
-    private val readerPostMoreButtonUiStateBuilder: ReaderPostMoreButtonUiStateBuilder
+    private val readerPostMoreButtonUiStateBuilder: ReaderPostMoreButtonUiStateBuilder,
+    private val readerPostTagsUiStateBuilder: ReaderPostTagsUiStateBuilder,
+    private val appPrefsWrapper: AppPrefsWrapper
 ) {
     // TODO malinjir move this to a bg thread
     fun mapPostToUiState(
         post: ReaderPost,
+        isDiscover: Boolean = false, // set to true for new discover tab
```

Comment:
@ashiagr - Does the future of `ReaderPostUiStateBuilder` make it possible to use on other tabs? Following, Saved, Likes, Automatic, etc. If so, I wonder if there will be any future logic directly related to what tab is shown and if so, perhaps implementing something other than a boolean may be an option. Feel free to ignore my suggestion, as boolean serves it purpose perfectly for our focus.

Reply:
We might want to convert it to tab name. We'll see how it turns out. Thanks for pointing it out :).